llvm/lib/Target/AMDGPU/SIInsertWaitcnts.cpp

```diff
@@ -388,6 +388,8 @@ class SIInsertWaitcnts : public MachineFunctionPass {
   // message.
   DenseSet<MachineInstr *> ReleaseVGPRInsts;
 
+  // bool insertWaitcntAfterMemOp(MachineFunction &MF);
```

Comment:
Remove all the unused code, don't just comment it out.

Reply:
Done.